script/update

```diff
@@ -300,6 +317,46 @@ def download(url, filename):
           break
         f.write(chunk)
 
+def get_packaged_src(clean, git_cache, nohooks, upload_src):
+  src_version = chromium_version()
+  tar_name = 'src.tar.bz2'
+  package_url = '{0}/{1}/src/{2}/{3}'.format(CHROMIUMCONTENT_SRC_URL,
+                                         PLATFORM_KEY, src_version, tar_name)
+  try:
+    print('Trying to download ' + package_url)
+    filesystem.download_and_extract(SOURCE_ROOT, package_url)
+  except urllib2.URLError as err:
+    print('Could not download {0}, syncing from git'.format(package_url))
+    gclient_sync(src_version, clean, git_cache, nohooks)
+    if upload_src:
+      print('Packaging source to ' + tar_name + '.')
+      tar_path =  os.path.join(SOURCE_ROOT, tar_name)
+      filesystem.safe_unlink(tar_path)
+      with util.scoped_cwd(SOURCE_ROOT):
+        tar_file = tarfile.open(tar_path, 'w:bz2')
+        tar_file.add('src', exclude=exclude_packaged_files)
+        tar_file.close()
+      if ('LIBCHROMIUMCONTENT_S3_ACCESS_KEY' in os.environ and
+         'LIBCHROMIUMCONTENT_S3_BUCKET' in os.environ and
+         'LIBCHROMIUMCONTENT_S3_SECRET_KEY' in os.environ):
+        script_path = os.path.join(SOURCE_ROOT, 'script', 'upload')
+        script_arguments = ['--upload_packaged_src', '--chromium_version', src_version]
+        print('Uploading source to ' + package_url + '.')
+        subprocess.check_call([sys.executable, script_path] + script_arguments)
+  except:
+    print('Unexpected error downloading and extracting chromium source:', sys.exc_info()[0])
+    raise
+
+
+def exclude_packaged_files(filename):
+  if ".git" in filename:
+    return True
+  else:
+    should_exclude = filename in SRC_FILES_TO_IGNORE
+    if not should_exclude:
+      print('Adding {0} to tarfile'.format(filename))
```

Comment:
Is this debugging output that's no longer necessary? Seems like this would print a LOT

Reply:
CircleCI has this thing where if it doesn't receive output in a certain amount of time, it will cancel the job.  I added this output here to avoid that.  I can probably remove it now that I know approximately how long it will take to run that bit so I can set an appropriate timeout

Reply:
Ah cool, sounds good!